flamedisx/inference.py

```diff
@@ -409,6 +469,7 @@ class IntervalObjective(Objective):
     # Add constant offset to objective, so objective is not 0 at the minimum
     # and relative tolerances mean something.
     _offset = 1
+    _callbackbag = []
```

Comment:
unused

Reply:
Thinking of adding a callback feature to record the orbit of the optimizer. `_callbackbag` is used to store these intermediate values of the optimizer but the callback function is currently broken.